src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheck.java

```diff
@@ -176,14 +180,19 @@ private static DetailAST getParent(DetailAST commentBlock) {
             result = commentBlock.getNextSibling();
         }
 
+        final List<Integer> typesArrayList = new ArrayList<>(Arrays.asList(
```

Comment:
If this is now a hardcoded list that never changes, then please move it to a static final field.

Reply:
Fixed.